content/cloud/linking-applications-to-resources-using-labels.md

```diff
@@ -0,0 +1,143 @@
+
+Now we have two seperate Spin applications (`todo-app` and `todo-app-v2`) using their own labels to link to the same database. 
+
+Links and labels are application scoped. Whenever a specific Spin application (say `todo-app`) needs to access the database resource (`inspiring-otter`), it will do so via the label "data".
+
+The Spin application does not need to know the true name of the Cloud resource (`inspiring-otter` database) that it is interacting with. 
+
+A label is understood by all the components within a single application, but the labels applied do not hold any significance to other Spin applications. 
+
+> At this time, only NoOps SQL DB supports labels. However, in the future other Fermyon Cloud resources such as Key Value Store will support links and labels as well. 
+
+## Benefits of Links and Labels
+
+Benefits of the Fermyon Clould links and labels include:
+
+* **Easy Sharing**: Share your Fermyon Cloud resource across applications effortlessly.
+* **Resource Creation Control**: You decide when to create and delete Fermyon Cloud resources.
+* **Flexible Lifecycle Management**: Manage Fermyon Cloud resources independently of your Spin application.
+* **Seamless Cloud Integration**: Cloud experience smoothly integrates with local Spin development, no runtime configuration changes required. 
+* **Dynamic App Resource Configuration**: Change the data resource your application points to without rebuilding it.
+
+## Working With Links and Labels
+
+Whenever you add a string to your NoOps SQL database entry in the component manifest, you're creating a label. When you deploy your Spin application to Fermyon Cloud, you will be prompted to link the label to a specific NoOps SQL database instance. Later on, if you'd like to update the link you can do so via the `spin cloud link` command. 
+
+To see your existing labels, you will need to use the `spin cloud` plugin along with the subcommand of the resource you've linked your Spin application to (via a label). For example, if you would like to see the labels you created for your NoOps SQL DB resources, you would run the following command:
+
+```bash
+$ spin cloud sqlite list
++--------------------------------------------------------+
+| App                          Label     Database        |
++========================================================+
+| todo-app                     data      inspiring-otter |
+| todo-app-v2                  default   inspiring-otter |
++--------------------------------------------------------+
+```
+
+Now you can see your Spin applications, their respective labels and their connected NoOps SQL Databases.
+
+If you'd like to change your link while your Spin application is running, you can do so with the following command:
+
+```bash
+spin cloud sqlite unlink --app todo-app data
```

Comment:
Er... this doesn't change it... this breaks it... now `todo-app` is going to start failing.  You really only want to `unlink` when the application is offline.